### ISSUE TITLE
DMN Unique Hit Policy table has a bug

### ISSUE THREAD

**Issue by jwulf** [camunda/camunda-docs#823]
In the [DMN Unique Hit Policy example table](https://docs.camunda.io/docs/components/modeler/dmn/decision-table-hit-policy/#unique), "Summer" is missing and "Spring" is in twice.

Also, it is unclear in the docs what happens when more than one rule *could* match. 

For example, if I have a discriminated union, as in the example, it is clear that at most one rule match is possible. However, what if I have something like this:

Summer and temperature > 20
Summer and temperature < 22

In this case, Summer and a temperature of 21 will match both rules. Does UNIQUE mean that it will match the first one in the table?

**Comment by akeller** [comment]
@saig0 can you take a look at this?

**Comment by saig0** [comment]
@jwulf good point :laughing: Please note that we just copied the [content from C7](https://docs.camunda.org/manual/7.17/reference/dmn/decision-table/hit-policy/#unique-hit-policy). 
It seems that you're the first one who really reads the docs :sweat_smile:

I would like to improve the DMN docs but it's a bigger effort. Currently, we don't plan to work on the docs soon. 
However, contributions are welcome :wink: 

Regarding your question:
> Does UNIQUE mean that it will match the first one in the table?
